recipes/rb-tty-prompt/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "tty-prompt" %}
+{% set version = "0.19.0" %}
+
+package:
+  name: rb-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://rubygems.org/downloads/{{ name }}-{{ version }}.gem
+  sha256: 3347d3535aaabd71952e52b0936f523ce77d0b345b1649b733dd9a9a92d8e157
+
+build:
+  noarch: generic
+  number: 0
+  script:
+    - gem install --norc -l -V --ignore-dependencies {{ name }}-{{ version }}.gem
+      && gem unpack {{ name }}-{{ version }}.gem
+  skip: {{ win }}
+
+requirements:
+  host:
+    - ruby
+  run:
+    - ruby
+    - rb-necromancer >=0.5,<1
+    - rb-pastel >=0.7.3,<1
```

the constraint was `~>0.7.0`
```suggestion
    - rb-pastel >=0.7.0,<0.8
```

---

Thanks very much. I have committed your requested changes.

---

i still seem to be getting a version error with `rb-pastel >=0.7.0,<0.8`